Java/JDI/jdi-cucumber-test-generator/src/main/java/com/epam/test_generator/DozerMapper.java

```diff
@@ -0,0 +1,22 @@
+package com.epam.test_generator;
+
+import org.dozer.DozerBeanMapper;
+import org.dozer.Mapper;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DozerMapper {
+
+    private static Mapper mapper;
+
+    static {
+        List cfg = new ArrayList();
+        cfg.add("dozerMapping.xml");
+        mapper = new DozerBeanMapper(cfg);
+    }
+
+    public static void map(Object source, Object dest) {
```

Comment:
Why it is `static`? Let's make it an object method instead. This would allow us to use Spring for injecting this mapper.